ViewModel/LibraryViewModel.swift

```diff
@@ -156,7 +156,7 @@ final class LibraryViewModel: ObservableObject {
                     return
                 }
             }
-            let parser = try await parse(data: data)
+            let parser = try await parse(data: data, urlHost: responseURL)
```

Comment:
I find `urlHost` (variable name) misleading as that's the same name as what we pass to libkiwix but it represent a different URL